.github/workflows/on_release_publish.yml

```diff
@@ -15,13 +15,13 @@ jobs:
     - name: Check out code
       uses: actions/checkout@v2
 
-    # on release we want to use release.tag_name for the version
+    # on release, we want to use release.tag_name for the version
     - name: Set tegola version (use release.tag_name)
       if: github.event_name == 'release'
       run: echo ${{ github.event.release.tag_name }} > ${{ github.workspace }}/version.txt
 
     # when it's not a release build use the commit hash for the version tag
-    - name: Set tegola version (use commit hash)
+    - name: Set Tegola version (use commit hash)
```

Comment:
why capitalize `Tegola`? it's not capitalized anywhere else.

Reply:
More force of habit. Will revert that back.